x-pack/plugins/actions/server/builtin_action_types/cases_webhook/validators.ts

```diff
@@ -59,4 +69,59 @@ export const validateSecrets = (secrets: CasesWebhookSecretConfigurationType) =>
 export const validate: ExternalServiceValidation = {
   config: validateConfig,
   secrets: validateSecrets,
+  connector: validateConnector,
+};
+
+const validProtocols: string[] = ['http:', 'https:'];
+export const assertURL = (url: string) => {
+  try {
+    const parsedUrl = new URL(url);
+
+    if (!parsedUrl.hostname) {
+      throw new Error('URL must contain hostname');
+    }
+
+    if (!validProtocols.includes(parsedUrl.protocol)) {
+      throw new Error('Invalid protocol');
+    }
+  } catch (error) {
+    throw new Error(`${error.message}`);
```

Comment:
Can we include the url in the error message?

Reply:
https://github.com/elastic/kibana/pull/131762/commits/3d4fcc41ea8df3a0162b9e3695b938bd5f21b5ae